### ISSUE TITLE
Error when running app: Failed to set ([prop]) user defined inspected property on (UILabel)

### ISSUE THREAD

**Issue by zomars** [sendbird/sendbird-uikit-ios-sources#7]
## Steps followed

1. Cloned the repo
2. `cd uikit-sample`
3. `pod install`
4. Changed application Id on `uikit-sample/SendBirdUIKit-Sample/AppDelegate.swift`
5. Run app on simulator `iPhone 12 mini`
6. Got this error on console:

```
2021-08-13 11:46:43.802507-0600 SendBirdUIKit-Sample[90836:1717383] [] nw_protocol_get_quic_image_block_invoke dlopen libquic failed
2021-08-13 11:46:43.811488-0600 SendBirdUIKit-Sample[90836:1717217] Failed to set (top) user defined inspected property on (UILabel): [<UILabel 0x7f858b621940> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key top.
2021-08-13 11:46:43.811744-0600 SendBirdUIKit-Sample[90836:1717217] Failed to set (bottom) user defined inspected property on (UILabel): [<UILabel 0x7f858b621940> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key bottom.
2021-08-13 11:46:43.812007-0600 SendBirdUIKit-Sample[90836:1717217] Failed to set (leading) user defined inspected property on (UILabel): [<UILabel 0x7f858b621940> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key leading.
2021-08-13 11:46:43.812299-0600 SendBirdUIKit-Sample[90836:1717217] Failed to set (trailing) user defined inspected property on (UILabel): [<UILabel 0x7f858b621940> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key trailing.
```

## What I've tried

- running `pod update`
- setting `platform :ios, '13.0'` on Podfile and reinstalling
- Removing `PaddingLabel.swift` and it's references.

Any help is appreciated

**Comment by tezpark** [comment]
Hello @zomars,

The below issue only occurs on the simulator and does not affect the app, so you don't have to worry about it.
> ```
> [] nw_protocol_get_quic_image_block_invoke dlopen libquic failed
> ```
> 


And the problem below is that unnecessary properties remain in the storyboard file, so I will remove them.
> ```
> Failed to set (bottom) user defined inspected property on (UILabel): [<UILabel 0x7f858b621940> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key bottom.
> Failed to set (leading) user defined inspected property on (UILabel): [<UILabel 0x7f858b621940> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key leading.
> Failed to set (trailing) user defined inspected property on (UILabel): [<UILabel 0x7f858b621940> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key trailing.
> ```
> 

If you report UIKit issues to the community, you can get help faster and more in detail.
[sendbird-chat-uikit community](https://community.sendbird.com/c/sendbird-chat-uikit)

**Comment by zomars** [comment]
Thank you, I can confirm it's solved in 0bbfb6b
